example/transform/huggingface_pdf_source_irs_QA.ipynb

```diff
@@ -0,0 +1,2015 @@
+      "Requirement already satisfied: marshmallow<4.0.0,>=3.18.0 in /opt/conda/envs/uniflow/lib/python3.10/site-packages (from dataclasses-json<0.7,>=0.5.7->langchain) (3.20.1)\n",
+      "Requirement already satisfied: typing-inspect<1,>=0.4.0 in /opt/conda/envs/uniflow/lib/python3.10/site-packages (from dataclasses-json<0.7,>=0.5.7->langchain) (0.9.0)\n",
+      "Requirement already satisfied: jsonpointer>=1.9 in /opt/conda/envs/uniflow/lib/python3.10/site-packages (from jsonpatch<2.0,>=1.33->langchain) (2.4)\n",
+      "Requirement already satisfied: anyio<5,>=3 in /opt/conda/envs/uniflow/lib/python3.10/site-packages (from langchain-core<0.2,>=0.1->langchain) (4.2.0)\n",
+      "Requirement already satisfied: packaging<24.0,>=23.2 in /opt/conda/envs/uniflow/lib/python3.10/site-packages (from langchain-core<0.2,>=0.1->langchain) (23.2)\n",
+      "Requirement already satisfied: annotated-types>=0.4.0 in /opt/conda/envs/uniflow/lib/python3.10/site-packages (from pydantic<3,>=1->langchain) (0.6.0)\n",
+      "Requirement already satisfied: pydantic-core==2.14.6 in /opt/conda/envs/uniflow/lib/python3.10/site-packages (from pydantic<3,>=1->langchain) (2.14.6)\n",
+      "Requirement already satisfied: typing-extensions>=4.6.1 in /opt/conda/envs/uniflow/lib/python3.10/site-packages (from pydantic<3,>=1->langchain) (4.9.0)\n",
+      "Requirement already satisfied: six>=1.5 in /opt/conda/envs/uniflow/lib/python3.10/site-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)\n",
+      "Requirement already satisfied: charset-normalizer<4,>=2 in /opt/conda/envs/uniflow/lib/python3.10/site-packages (from requests<3,>=2->langchain) (3.3.2)\n",
+      "Requirement already satisfied: idna<4,>=2.5 in /opt/conda/envs/uniflow/lib/python3.10/site-packages (from requests<3,>=2->langchain) (3.6)\n",
+      "Requirement already satisfied: urllib3<3,>=1.21.1 in /opt/conda/envs/uniflow/lib/python3.10/site-packages (from requests<3,>=2->langchain) (2.1.0)\n",
+      "Requirement already satisfied: certifi>=2017.4.17 in /opt/conda/envs/uniflow/lib/python3.10/site-packages (from requests<3,>=2->langchain) (2023.11.17)\n",
+      "Requirement already satisfied: greenlet!=0.4.17 in /opt/conda/envs/uniflow/lib/python3.10/site-packages (from SQLAlchemy<3,>=1.4->langchain) (3.0.3)\n",
+      "Requirement already satisfied: sniffio>=1.1 in /opt/conda/envs/uniflow/lib/python3.10/site-packages (from anyio<5,>=3->langchain-core<0.2,>=0.1->langchain) (1.3.0)\n",
+      "Requirement already satisfied: exceptiongroup>=1.0.2 in /opt/conda/envs/uniflow/lib/python3.10/site-packages (from anyio<5,>=3->langchain-core<0.2,>=0.1->langchain) (1.2.0)\n",
+      "Requirement already satisfied: mypy-extensions>=0.3.0 in /opt/conda/envs/uniflow/lib/python3.10/site-packages (from typing-inspect<1,>=0.4.0->dataclasses-json<0.7,>=0.5.7->langchain) (1.0.0)\n"
+     ]
+    }
+   ],
+   "source": [
+    "!{sys.executable} -m pip install langchain pandas pypdf"
```

Comment:
can you change this cell to `!{sys.executable} -m pip install -q langchain pandas pypdf` and rerun this cell? this output is too long and distractive.